base_layer/wallet/src/storage/sqlite_db/wallet.rs

```diff
@@ -104,6 +104,82 @@ impl Argon2Parameters {
     }
 }
 
+/// A structure to hold key-related database field data, to make atomic operations cleaner
+pub struct DatabaseKeyFields {
```

Comment:
```suggestion
pub struct DatabaseEncryptionFields {
```
NIT: Just reading `DatabaseKeyFields` does not tell me what it is, to me it could be anything from database Max_read/write settings, to a key value store.